### PR TITLE
Test fixes, export to CSV refactoring

### PR DESCRIPTION
**Pull request by bamse16**
I don't know how to create a separate pull request 🙂

### CODE REVIEW

File: BodyweightFitness/AbstractViewController.swift [hunk 1]
```diff
@@ -53,7 +53,7 @@ class AbstractViewController: UIViewController {
     func createBackgroundView(height: Int = 50) {
         let view = UIView()
         view.backgroundColor = UIColor.primary()
-        
+        self.contentView.backgroundColor = UIColor.white
```
Comment (mazurio) [inline BodyweightFitness/AbstractViewController.swift]: This can be deleted :-)

File: Podfile [hunk 2]
```diff
@@ -34,7 +34,9 @@ post_install do |installer|
     if ['Charts'].include? target.name
       target.build_configurations.each do |config|
         config.build_settings['SWIFT_VERSION'] = '4.0'
+        end
```
Comment (mazurio) [inline Podfile]: Is this `end` needed here?

File: BodyweightFitness/Main/HomeViewController.swift [hunk 3]
```diff
@@ -10,6 +10,7 @@ class HomeViewController: AbstractViewController, MFMailComposeViewControllerDel
     override func viewDidLoad() {
         super.viewDidLoad()
         
+        self.view.backgroundColor = UIColor.primary()
```
Comment (mazurio) [inline BodyweightFitness/Main/HomeViewController.swift]: Is this needed here?

**Review by mazurio — COMMENTED** [review]
Just some general comments.

File: BodyweightFitness/AbstractViewController.swift (hunk 1 above)
Comment (bamse16) [reply BodyweightFitness/AbstractViewController.swift]: Fixed, I'm going to send a new pull request

**Comment by bamse16** [comment]
Fixed previous comments.

**Review by mazurio — APPROVED** [review]

**Comment by mazurio** [comment]
👍 Thanks!

**Pull request merged**